sixMans/queue.py

```diff
@@ -1,13 +1,14 @@
-import discord
 import collections
 import datetime
 import uuid
 from queue import Queue
+from typing import List
```

Comment:
What does this accomplish? Just curious.

Reply:
It just makes the variables strongly typed so we can guarantee they're the type we expect them to be. It also makes it a bit easier to reference them elsewhere and use the inherited methods/data.